### DISCUSSION TITLE
Can We Change The Preset Settings On The Loop?

### DISCUSSION THREAD

**Discussion by 23426356587** [mrkai77/Loop#457] [Q&A] [answered]
Can we change the preset settings on the loop? 

For example, instead of top left corner can I change the setting to go 2/3 left?

**Comment by SenpaiHunters** [comment]
Yes you can, in Settings > Keybindings  -- You can also sent custom snaps. Refer to the two provided images -- If however, you mean for the Radial Menu, to my knowledge, i don't think you can?

---

![Loop - June 06 2024 - 00 32 AM@2x](https://github.com/MrKai77/Loop/assets/103985728/f8ff9264-99db-404c-8270-e0f9aeeb1f6e)

![Loop - June 06 2024 - 00 33 AM@2x](https://github.com/MrKai77/Loop/assets/103985728/e6038752-06cc-434f-9554-75962a5e2323)

**Reply by 23426356587** [reply]
Thanks for the reply and screen shots.

I should have been more clear that I meant if you can edit the specific items in the radial menu. I love the UI for this tool because I'm looking to get away from keyboard shortcuts for screen manipulation.

**Reply by SenpaiHunters** [reply, marked as answer]
Ah, yea, this is not yet possible but it's in talks 😄

**Reply by deev1207** [reply]
> Yes you can, in Settings > Keybindings -- You can also sent custom snaps. Refer to the two provided images -- If however, you mean for the Radial Menu, to my knowledge, i don't think you can?
> 
> ![Loop - June 06 2024 - 00 32 AM@2x](https://private-user-images.githubusercontent.com/103985728/342379301-f8ff9264-99db-404c-8270-e0f9aeeb1f6e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAyMTk4ODEsIm5iZiI6MTc0MDIxOTU4MSwicGF0aCI6Ii8xMDM5ODU3MjgvMzQyMzc5MzAxLWY4ZmY5MjY0LTk5ZGItNDA0Yy04MjcwLWUwZjlhZWViMWY2ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyMlQxMDE5NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MTBmYTEyYTAzNWFjZTJkYTRmZmU3ZGM4M2FiOTZjNzUxNTdhYzY2ZmM0YTA2OTIzYjFjMzU1ZmJmMzI5YzY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Rqk-K3KIB4t0_RNx7ARyGueIK_1pKMC7M_mH78O9X1c)
> 
> ![Loop - June 06 2024 - 00 33 AM@2x](https://private-user-images.githubusercontent.com/103985728/342379272-e6038752-06cc-434f-9554-75962a5e2323.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAyMTk4ODEsIm5iZiI6MTc0MDIxOTU4MSwicGF0aCI6Ii8xMDM5ODU3MjgvMzQyMzc5MjcyLWU2MDM4NzUyLTA2Y2MtNDM0Zi05NTU0LTc1OTYyYTVlMjMyMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIyMlQxMDE5NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZjdkMDg5NTYwMzQ0ODNkODNiZTA2MTkzZjU2NDNhZWE0ZjlhMjdhYzg1NWZiZjkwZTIxMDZhZjg0MTUyNWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.A94dmzcmbFZaGkMkVeTHisr1BIou0OreYfbiMtoS4JU)

thanks for this, I was trying to search how to do custom snaps but couldn't find it easily. A recommendation would be to add it your readme or somewhere more visible.
